openedx/features/student_certificates/tests/test_views.py

```diff
@@ -49,7 +50,9 @@ def setUpClass(cls):
         from openedx.features.course_card.tests.helpers import initialize_test_user
         cls.user = initialize_test_user(password=cls.password)
         cls.user.save()
-        cls.course_settings = CustomSettings(id=cls.course.id, course_short_id=1, course_open_date=get_current_utc_date())
+        cls.course_settings = CustomSettings(id=cls.course.id,
+                                             course_short_id=1,
+                                             course_open_date=get_current_utc_date())
```

Comment:
Let's please follow the edx [recommendations](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines) regarding breaking long lines.